RxBlocking/RunLoopLock.swift

```diff
@@ -84,6 +84,8 @@ final class RunLoopLock {
                     return
                 case .timedOut:
                     throw RxError.timeout
+                @unknown default:
```

Comment:
Will this actually work in Swift 4? 🤔

Reply:
I'm not sure how far back it works, but it at least works with 4.2. We'll see how CI feels about it

Reply:
Looks like not in Xcode 9.3, I'll gate it some how

Reply:
Seems like just `default` might work here